pkg/registry/pod/strategy.go

```diff
@@ -31,11 +31,39 @@ import (
 	"k8s.io/kubernetes/pkg/fields"
 	"k8s.io/kubernetes/pkg/labels"
 	"k8s.io/kubernetes/pkg/registry/generic"
+	nodeetcd "k8s.io/kubernetes/pkg/registry/node/etcd"
 	"k8s.io/kubernetes/pkg/runtime"
 	"k8s.io/kubernetes/pkg/util"
 	"k8s.io/kubernetes/pkg/util/fielderrors"
 )
 
+type HostLocator interface {
```

Comment:
Well, if I move it to the Node package I'll need to import it as well. But yes - probably it's a better place for it.